### ISSUE TITLE
Material 3 Demo running with Skwasm doesn't render all elements on first paint

### ISSUE THREAD

**Issue by eyebrowsoffire** [flutter/samples#2008]
When running the Material 3 Demo with Skwasm, some of the elements in the left bar (text, icons, and some of the images) don't seem to render on first paint. Usually after resizing the window or doing something else that forces a redraw, it recovers.

**Comment by parlough** [comment]
Is this something you suspect this is something the app itself is doing wrong even not in the context of skwasm? Otherwise it sounds like this issue is better tracked on `flutter/flutter`

**Comment by eyebrowsoffire** [comment]
Yes, I don't know why I originally filed this here, I definitely should have filed it against `flutter/flutter`
